src/components/lockPage.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
+import { FiLock, FiUnlock} from 'react-icons/fi'
+import { FaBatteryHalf, FaBatteryQuarter, FaBatteryThreeQuarters, FaBatteryFull} from 'react-icons/fa'
+import bettan from '../images/bettan.png'
+import configura from '../images/configura-white.png'
+
+import { success, error, logoContainer, buttons, roomTitle, batteryContainer} from '../scss/lock.module.scss'
+import { post } from './request'
+import BigPixels from './layout/bigPixels'
+import { IconButton } from './ui/buttons'
+import { GridContainer, GridItem } from './ui/grid'
+import useSWR from 'swr'
+
+const STATUS_REFRESH_INTERVAL = 180 * 1000 // 3 minutes
+const CRITICAL_BATTERY_LEVEL = 15;
+
+const BatteryIcon = ({batteryLevel}) => {
+  return <FaBatteryFull></FaBatteryFull>
+  if (batteryLevel > 75) return <FaBatteryFull />;
+  if (batteryLevel > 50) return <FaBatteryThreeQuarters />;
+  if (batteryLevel > 25) return <FaBatteryHalf />;
+  if (batteryLevel > 0) return <FaBatteryQuarter />;
```

Comment:
Inga av IF-satserna kommer att kollas. För det är en return på rad 18.
Funderar på om man ska ha en ikon för om batterinivån är 0?

```suggestion
  if (batteryLevel > 75) return <FaBatteryFull />;
  if (batteryLevel > 50) return <FaBatteryThreeQuarters />;
  if (batteryLevel > 25) return <FaBatteryHalf />;
  if (batteryLevel > 0) return <FaBatteryQuarter />;
  if (batteryLevel == 0) return <FaBatteryEmpty />;
```